tests/e2e/lba_procedures/read_permission_queries.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import pytest
+import sys
+
+from typing import List
+
+from common import connect, execute_and_fetch_all, reset_permissions
+
+match_query = "MATCH (n) RETURN n;"
+match_by_id_query = "MATCH (n) WHERE ID(n) >= 0 RETURN n;"
+
+match_by_label_query = "MATCH (n:read_label) RETURN n;"
+match_by_label_property_range_query = "MATCH (n:read_label) WHERE n.prop < 7 RETURN n;"
+match_by_label_property_value_query = "MATCH (n:read_label {prop: 5}) RETURN n;"
+match_by_label_property_query = "MATCH (n:read_label) WHERE n.prop IS NOT NULL RETURN n;"
+
+
+def get_read_node_without_index_operation_cases() -> List[str]:
```

Comment:
```suggestion
def get_read_node_without_index_operation_cases() -> List[List[str]]:
```

Reply:
And why not make these just variables, that you can use in the test immediately

Reply:
Okay, makes sense, will make them into global variables